components/blitz/src/ome/services/blitz/repo/InternalRepositoryI.java

```diff
@@ -20,7 +20,7 @@
 /**
  * Standalone repository service.
  *
- * @DEV.TODO Better named "StandaloneRepositoryI"
+ * TODO Better named "StandaloneRepositoryI"
```

Comment:
Think @mtbc moved these to `//`-style comments so they don't show up in the javadoc.